tapcfg/server.go

```diff
@@ -423,6 +429,7 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 		UniverseQueriesBurst:     cfg.Universe.UniverseQueriesBurst,
 		RfqManager:               rfqManager,
 		AuxLeafCreator:           auxLeafCreator,
+		AuxLeafSigner:            auxLeafSigner,
```

Comment:
We're always creating and starting these components. Since they're pretty light-weight, it IMO doesn't make sense to only conditionally create and start them for the custom channel specific use case, as that would mean we'd need to deal with more `.WhenSome()` and `.IsSome()` constructs.

But maybe something to consider in this regard is to only start the HTLC and invoice acceptors if they're really needed, since both of them can only be registered once in `lnd`. So if we always occupy that single "slot" with `tapd` even if you don't use custom channels, that could create issues for some users. I'll look into that in the invoice PR.